arches/app/models/models.py

```diff
@@ -2383,6 +2385,134 @@ def __str__(self):
     class Meta:
         managed = True
         db_table = "spatial_views"
+        triggers = [
+            pgtrigger.Composer(
+                name="__arches_trg_update_spatial_views",
+                when=pgtrigger.After,
+                operation=pgtrigger.Update | pgtrigger.Delete | pgtrigger.Insert,
+                declare=[
+                    ("sv_perform", "text"),
+                    ("valid_geom_nodeid", "boolean"),
+                    ("has_att_nodes", "integer"),
+                    ("valid_att_nodeids", "boolean"),
+                    ("valid_language_count", "integer"),
+                ],
+                func=pgtrigger.Func(
```

Comment:
In arches-for-science, we worked around this by putting the SQL string in a [separate file](https://github.com/archesproject/arches-for-science/blob/4bc234956b3e1b3c3d4b6767202f198421a58203/arches_for_science/trigger_functions.py#L4): I'd suggest trying that here. Having the trigger wrapper here still provides some "discoverability" IMO without having to have the monster string right here.

Reply:
much nicer, thanks for the suggestion.